site/en/r2/tutorials/generative/dcgan.ipynb

```diff
@@ -1,143 +1,197 @@
 {
```

Comment:
<p>"Except the output layer which uses"</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Thanks, fixed.</p>